src/client/pages/OfferNew/Introduction/index.tsx

```diff
@@ -1,56 +1,74 @@
+import React, { useState } from 'react'
 import styled from '@emotion/styled'
 import { colorsV3 } from '@hedviginsurance/brand'
-import React from 'react'
-import { Container, Section } from 'pages/OfferNew/components'
+import { Section } from 'pages/OfferNew/components'
 import { OfferData } from 'pages/OfferNew/types'
-import { isBundle } from 'pages/OfferNew/utils'
-import { useDocumentScroll } from '../../../utils/hooks/useDocumentScroll'
-import { ExternalInsuranceProvider } from './ExternalInsuranceProvider'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { HeroOfferDetails } from './HeroOfferDetails'
 import { Sidebar } from './Sidebar'
 
-import { Usps } from './USPS'
-
-interface Props {
+type Props = {
   offerData: OfferData
   refetch: () => Promise<void>
   onCheckoutOpen: () => void
 }
 
+type HeroImageProps = {
+  hasLoaded: boolean
+}
+
+const HERO_HEIGHT = '400px'
+
 const Wrapper = styled.div`
   width: 100%;
-  padding: 7rem 0;
-  background-color: ${colorsV3.gray900};
-  position: relative;
-  box-sizing: border-box;
+  height: ${HERO_HEIGHT};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
 
-  @media (max-width: 1020px) {
-    padding: 7rem 0 7rem 0;
-  }
+const Hero = styled.div`
+  width: 100vw;
+  height: ${HERO_HEIGHT};
+  background: ${colorsV3.gray900};
+  overflow: hidden;
+  position: absolute;
+`
+
+const HeroImage = styled.img<HeroImageProps>`
+  height: 1000px;
+  object-fit: cover;
+  object-position: -550px -100px;
+  opacity: ${({ hasLoaded }) => (hasLoaded ? 0.5 : 0)};
+  transition: opacity 0.8s;
```

Comment:
Should we have some kind of easing, like `ease-out`?

Reply:
Sure, I can add that. Believe it or not - I don't have a strong opinion here! 😄

Reply:
But interestingly the default is apparently `ease` so it's not linear as it is now. Would you rather have `ease-out` than `ease` though?

Reply:
Aha nice, my gut feeling was that it was `linear`, that makes sense though! I tend to use `ease-out` for fading in stuff. I usually have a sneak peek on how material design resonates about easings which I find helpful 🤗https://material.io/archive/guidelines/motion/duration-easing.html#duration-easing-natural-easing-curves